lnwallet/channel.go

```diff
@@ -3892,16 +3892,30 @@ func (lc *LightningChannel) ProcessChanSyncMsg(
 			return nil, nil, nil, err
 		}
 
+		var commitUpdates []lnwire.Message
```

Comment:
I wonder if we could also encode the `OweCommitment` case above using this new key?

Reply:
If `OweCommitment` is called, then this block of code cannot be called

Reply:
Sorry, I don't understand the suggestion

Reply:
I think he means modify the logic above that determines if we need to send a new sig to use the new value added in the change. IMO we should just try to make the smallest change here, so we can isolate the impact of the diff itself.